### PR TITLE
Jun aggregate reports

### PR DESCRIPTION
**Pull request by perigee**
@bpostlethwaite  @scjody 

Spent time on learning `Go`, here is some `ugly` fix to the issue https://github.com/plotly/plotbot/issues/29

Compiled, and need to be tested

### CODE REVIEW

**Comment by perigee** [comment]
now `junbot` is living in channel [education-pages](https://plotly.slack.com/messages/education-pages/) for your testing

**Comment by bpostlethwaite** [comment]
Great! I'll review and test tonight. I like JunBot, brings to mind June Bug

**Comment by bpostlethwaite** [comment]
also invited you into the bottest room. Though testing bots in education-pages is pretty genius

**Comment by bpostlethwaite** [comment]
Could we not print the report template for a repo if the repo has no bugs? This will almost always be the case in Deployment. Also can we add https://github.com/plotly/plotly.py

Why did I use channels when writing this.... probably just to learn channels... **sigh**

File: bugger/bugger.go [hunk 1]
```diff
@@ -57,6 +52,39 @@ func (bugger *Bugger) makeBugReporter(days int) (reporter bugReporter) {
 	return
 }
 
+func (bugger *Bugger) aggregateBugReporter(conv *plotbot.Conversation, msg *plotbot.Message) {
+	if len(bugger.ghclient.Conf.Repos) == 0 {
+			log.Println("No repos configured - can't produce a bug report")
```
Comment (scjody) [inline bugger/bugger.go]: Have you run `gofmt` on this recently?

**Comment by scjody** [comment]
Overall this looks good.

Note that the deployment repo as configured will *never* have issues in it, so it should be left out of Plotbot.

**Comment by perigee** [comment]
@scjody @bpostlethwaite  modified code accordingly, the `junbot` is living in channel [bottest](https://plotly.slack.com/messages/bottest/), the `deployment` repo left there for showing the case for no bug during period, will be removed from configuration file after testing.

**Comment by scjody** [comment]
OK, :dancer: from me.

In future, please perform formatting and logic changes as **separate commits**.

**Comment by bpostlethwaite** [comment]
@scjody there are two places where tabs are used instead of spaces which I assume is a change over of editor defaults from contributors. Would you actually manually for them to remain spaces, commit code, then allow go-fmt to turn them into tabs? Seems like over kill IMHO

**Comment by perigee** [comment]
@bpostlethwaite, sorry, quick figures, could you indicate me the lines? I used `gofmt` directly to correct the format.

**Comment by bpostlethwaite** [comment]
ah it's just two lines with spaces -> tabs which IS the correct way to gofmt code. You are all good. I was just curious about how absolute @scjody is with formatting vs logic commits. I understand when it makes a mess in the file. But in this case it hardly seems to warrant commenting on.

**Comment by scjody** [comment]
@bpostlethwaite I agree it's not a big deal in this case (although it's more than 2 lines), but it's easy to do this right if you're aware of the issue.  (You can `gofmt` manually then commit, or use `git commit --patch`.)

**Pull request merged**